app/configurator/components/add-new-dataset-panel.tsx

```diff
@@ -78,20 +78,28 @@ export const AddNewDatasetPanel = () => {
   const [dataSetIri, setDataSetIri] = useState("");
   const addNewDataset = useAddChartConfigBasedOnNewDataset();
 
+  const handleClose = useEventCallback(() => {
+    close();
+    setTimeout(() => {
+      // Give time to the drawer to close before resetting the dataset IRI
+      setDataSetIri("");
+    }, 500);
+  });
```

Comment:
Nit: Wondering if this shouldn't be in sync with (I assume) framer's transition duration on drawer close?

Reply:
Thanks for your vigilance, I used the SlideProps.onExited callback for this now :)